src/components/UnitCard.js

```diff
@@ -12,27 +16,35 @@ const UpgradeBar = styled.div`
   display: flex;
 `;
 
-const RemoveButton = styled.button`
+const RemoveButton = styled.div`
```

Comment:
This actually has nothing to do with the remove button and everything to do with its positioning. If you were just looking at the JSX below, you'd generally assume that there was more going on with this component and in this case, you'd actually expect it to be a button and not a div.